lib/resque.rb

```diff
@@ -165,6 +165,12 @@ def prune_interval
     @prune_interval || DEFAULT_PRUNE_INTERVAL
   end
 
+  attr_writer :enqueue_front
+  def enqueue_front
+    return @enqueue_front unless @enqueue_front.nil?
+    @enqueue_front = false
+  end
```

Comment:
Yea, it will be `nil`. But in this case (`enqueue_front ? ... : ...`) it will do the same thing. Anyway, it's up to you.